social_django/storage.py

```diff
@@ -152,7 +152,7 @@ def store(cls, server_url, association):
         except cls.DoesNotExist:
             assoc = cls(server_url=server_url,
                         handle=association.handle)
-        assoc.secret = base64.encodestring(association.secret)
+        assoc.secret = base64.encodestring(association.secret).decode()
```

Comment:
Since Python 3.1, This should be `base54.encodebytes` to avoid a `DeprecationWarning`.

Reply:
But that does not work on Python 2...

Reply:
@claudep can you if/else or try/except depending on the version of Python this is running on ? 

@reviewers: if this is the only blocker to including this patch I will be happy to cherry-pick that and apply a change for Python 2 (I am running on Python 3 but I need this fix). Let me know how you want to proceed.